tests/cli.rs

```diff
@@ -144,6 +144,25 @@ fn list_mutants_in_factorial_json() {
         .assert_insta("list_mutants_in_factorial_json");
 }
 
+#[test]
+fn list_mutants_in_meta_tree() {
```

Comment:
To avoid the need for duplicating these tests, I've added tests that run `--list` in every testdata tree, in 6a21091. If you rebase onto main and then run `cargo insta test --review` it should fix this up.